charts/netbox/ci/extras-values.yaml

```
@@ -0,0 +1,30 @@
+extraConfig:
+  - values:
+    DUNGEON: synth
+    QUEEN: "will rock you"
+  - configMap:
+    name: extraconfig-cm
+    items: []
+    optional: true
+  - secret:
+    name: extraconfig-secret
+    secretName: "not so secret"
+  - volumes:
+      - name: extraconfig-volume
+        emptyDir:
+          medium: Memory
+          sizeLimit: 2Mi
+
+extraVolumes:
+  - name: test-volume
+    emptyDir:
+      medium: Memory
+      sizeLimit: 2Mi
+extraVolumeMounts:
+  - name: test
```

```suggestion
  - name: test-volume
```
You cannot mount a volume that does not exist under the same name 😅